website/content/docs/enterprise/network-segments.mdx

```diff
@@ -52,3 +66,171 @@ among themselves, and use Serf to maintain a shared view of the members of the
 pool for different purposes, like finding a Consul server in a local cluster,
 or finding servers in a remote cluster. A **segmented** LAN Gossip Pool limits a
 group of agents to only connect with the agents in its segment.
+
+# Network Segments Configuration
+
+## Server Configuration
+
+Server agents are members of all segments and, therefore, provide a connection
+point across segments. The datacenter includes a `<default>` segment, as well as
```

Comment:
```suggestion
Server agents are members of all segments. The datacenter includes a `<default>` segment, as well as
```
This might be misleading as the servers do not facilitate routing gossip traffic between segments.